### PR TITLE
Modals updated from default to Fliplet standart

### PR DESCRIPTION
**Pull request by MaksymShokin**
@sofiiakvasnevska 

## Issue
https://github.com/Fliplet/fliplet-studio/issues/5642

## Description
Updated modals.

## Backward compatibility
This change is fully backward compatible.

### CODE REVIEW

File: js/interface.js [hunk 1]
```diff
@@ -554,12 +554,26 @@ $('#app')
...
+        if (!result) {
+          return;
+        }
+
+        try {
+          table.destroy();
+        } catch(e) {}
+
+        dataSourceEntriesHasChanged = false;
+        $('.data-save-updated').addClass('hidden');
```
Comment (squallstar) [inline js/interface.js]: why were these 3 lines below necessary?

Comment (MaksymShokin) [reply js/interface.js]: Those lines navigate user to all data sources. I had to duplicate them to make both flows (saved, and not saved) to work properly.

File: js/interface.js [hunk 2]
```diff
@@ -670,26 +684,38 @@ $('#app')
...
+
+          Fliplet.DataSources.connect(currentDataSourceId).then(function(source) {
+            _this.removeClass('disabled').text('Add new user');
+            return source.addUserRole({
+              userId: userId,
+              permissions: permissions
+            });
+          }).then(fetchCurrentDataSourceUsers, function(err) {
+            _this.removeClass('disabled').text('Add new user');
+            Fliplet.Modal.alert({ message: err.responseJSON.message});
```
Comment (tonytlwu) [inline js/interface.js]:
```suggestion
            Fliplet.Modal.alert({ message: err.responseJSON.message });
```

Comment (MaksymShokin) [reply js/interface.js]: fixed

**Pull request merged**